tests/foreman/ui/test_subscription.py

```diff
@@ -27,9 +27,9 @@
     VDC_SUBSCRIPTION_NAME,
     VIRT_WHO_HYPERVISOR_TYPES,
 )
-from robottelo.utils.manifest import clone
+from robottelo.utils.issue_handlers import is_open
```

Comment:
this seems to be an leftover

Reply:
@ogajduse ack, thanks for catching that.